src/graph.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, interactome_file, params={}, skip_checks=False):
                 self.interactome_dataframe = self.interactome_dataframe.groupby('temp').max().reset_index()[["source","target","cost"]]
             else: del self.interactome_dataframe['temp']
 
-        self.interactome_graph = nx.from_pandas_dataframe(self.interactome_dataframe, 'source', 'target', edge_attr=self.interactome_dataframe.columns[2:].tolist())
+        self.interactome_graph = nx.from_pandas_edgelist(self.interactome_dataframe, 'source', 'target', edge_attr=self.interactome_dataframe.columns[2:].tolist())
```

Comment:
Networkx changed these functions in version 2, which I believe we are now requiring (https://networkx.github.io/documentation/stable/release/release_2.0.html #2558)